### PR TITLE
Prepare for Next Major Version Release

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Summary of changes

- Update CHANGELOG to remove `(next-major-version)` to prepare for rebase of `next-major-version` into `master`
  - Add missing note in CHANGELOG
- Update README with note on beta

### Checklist

- ~[ ] Added a changelog entry~

- @mattwylder @jaxdesmarais

### CODE REVIEW

**Comment by mattwylder** [comment]
You're probably wondering what the "empty file" diff for `BTPayPalClient.m` is. Somehow that the [file permissions](https://github.com/community/community/discussions/6814) were changed to include "execute" in `next-major-version` and changed back to normal in `update-release-script-nmv`. We have no idea how this happened, but since the permissions in `update-release-script-nmv` are correct, we're letting it be 🤷

**Comment by scannillo** [comment]
Is something with the commit history wonky here?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Braintree iOS SDK
 
+## iOS v6 Beta
+Currently, version 6 of the iOS SDK is in Beta. For stable releases, please point to version 5 of the SDK. While preparing for general availability, we expect to make breaking changes in the beta releases.
```
Comment (scannillo) [inline README.md:4]: Nitpick - I think this might look a little weird above the badge icons + don't think we need a new section header.

Comment (scannillo) [reply README.md:4]:
```
v5 is the latest major version of Braintree iOS. To update from v4, see the [v5 migration guide](https://github.com/braintree/braintree_ios/blob/master/V5_MIGRATION.md).
```

These #s and migration guide links should also be updated.

Comment (scannillo) [reply README.md:4]: Maybe that is where we can edit to include v6 beta info.

**Comment by jaxdesmarais** [comment]
Closing since the commit history here got weird when we changed the base branch from `update-cocoapods-spm` to `next-major-version`. Will reopen a new PR with these changes and apply the comments here: https://github.com/braintree/braintree_ios/pull/905#discussion_r1044867176

**Pull request closed without merging**